src/pages/Overview/OverviewPage.tsx

```diff
@@ -176,12 +201,9 @@ class OverviewPage extends React.Component<OverviewProps, State> {
                         <Link to={`/graph/namespaces?namespaces=` + encodedNsName}>{ns.name}</Link>
                       </CardTitle>
                       <CardBody>
-                        <ListPageLink
-                          target={this.state.isApp ? TargetPage.APPLICATIONS : TargetPage.WORKLOADS}
-                          namespace={ns.name}
-                        >
-                          {nbItems === 1 && (this.state.isApp ? '1 Application' : '1 Workload')}
-                          {nbItems !== 1 && nbItems + (this.state.isApp ? ' Applications' : ' Workloads')}
+                        <ListPageLink target={targetPage} namespace={ns.name}>
+                          {nbItems === 1 && oneItemText}
+                          {nbItems !== 1 && nbItems + pluralText}
                         </ListPageLink>
```

Comment:
why not 
```js
{nbItems === 1 ? oneItemText : nbItems + pluralText } 
```
?

Reply:
good question, no idea :)
i'll change it